docs/source/index.rst

```diff
@@ -14,6 +14,7 @@ Contents
    Overview <overview>
    Inputs <inputs>
    Run a Sonata Simulation <sonata-simulation>
+   NGV <ngv>
```

Comment:
Shall we move it below after "Developing a Custom Neurodamus <development>" ?

Reply:
ok